### PR TITLE
Updated the code style

### PR DESCRIPTION
**Pull request by Dundo7**
Here you go guys :). 

I think this is all for code style. 

Should we change the `Not preffered` and `Preffered` to `WRONG` and `RIGHT`?
If you find any duplicates, please let me know.

### CODE REVIEW

**Review by goranbrl — APPROVED** [review]

**Comment by goranbrl** [comment]
Don't merge this one until the code style repo PR is merged, since it's private for now :)

**Review by nikolamajcen — APPROVED** [review]

**Review by jabou — APPROVED** [review]

**Pull request merged**
